docs/installation-aks.md

```diff
@@ -0,0 +1,407 @@
+helm install azure/svc-cat-etcd --set azure.storage.account=$env:AZURE_STORAGE_ACCOUNT --set azure.storage.key=$env:AZURE_STORAGE_KEY
+```
+
+Once completed, you should see several etcd pods running:
+
+```console
+$ kubectl get pods
+
+NAME                                                              READY     STATUS    RESTARTS   AGE
+etcd-operator-etcd-operator-etcd-backup-operator-6b697d96c95fgv   1/1       Running   0          11m
+etcd-operator-etcd-operator-etcd-operator-676764c476-n4ftv        1/1       Running   0          11m
+etcd-operator-etcd-operator-etcd-restore-operator-7c8d6879rgkjv   1/1       Running   0          11m
+svc-cat-etcd-5xtj4vlhx8                                   1/1       Running   0          1m
+svc-cat-etcd-chfwgmjdph                                   1/1       Running   0          47s
+svc-cat-etcd-jj87b2hmwg                                   1/1       Running   0          31s
+```
+
+You should also have an etcd service:
+
+```console
+$ kubectl get service
+NAME                          TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)             AGE
+etcd-restore-operator         ClusterIP   10.0.231.143   <none>        19999/TCP           14m
+kubernetes                    ClusterIP   10.0.0.1       <none>        443/TCP             56m
+svc-cat-etcd          ClusterIP   None           <none>        2379/TCP,2380/TCP   3m
+svc-cat-etcd-client   ClusterIP   10.0.2.237     <none>        2379/TCP            3m
+```
+
+The `svc-cat-etcd-client` service is what you will use to configure Service Catalog.
+
+### Install Service Catalog
+
+Once you have created an etcd cluster, it is time to install Service Catalog. You will use Helm to install Service Catalog. There are a few values you will need to override for your Service Catalog installation:
+
+* RBAC must be disabled (pending AKS support)
+* Embedded etcd must be disabled
+* You must point the installation at an external etcd.
+
+```console
+helm repo add svc-cat https://svc-catalog-charts.storage.googleapis.com
+helm install svc-cat/catalog --name catalog --namespace catalog \
+   --set rbacEnable=false \
+   --set apiserver.storage.etcd.useEmbedded=false \
+   --set apiserver.storage.etcd.servers=http://svc-cat-etcd-client.default.svc.cluster.local:2379
+```
+
+## Create Azure Redis Cache
+
+Open Service Broker for Azure uses Redis as a backing store for its state. We recommend using a managed Redis service, such as Azure Redis Cache. By default, Azure Redis Cache only keeps data in memory. For best results, you will want to use the Premium tier in order to configure backups of the Redis data. Please see [How to configure data persistence for a Premium Azure Redis Cache](https://docs.microsoft.com/en-us/azure/redis-cache/cache-how-to-premium-persistence) for instructions on how to create an Azure Redis Cache. Once created, you can obtain the hostname and keys from the Portal or via the CLI. 
```

Comment:
"We recommend using Azure Redis Cache, a managed Redis service". Let's come right out and say that we commend out own stuff. 😀

Reply:
"By default, Azure Redis Cache ... For best results, "

As long as we are making recommendations let's be crystal clear

"You should use the Premium tier of Azure Redis Cache, so take advantage of persistent backups because by default Redis is an in-memory only database. See How to configure..."